webapp/src/ts/services/cht-script-api.service.ts

```diff
@@ -27,16 +32,33 @@ export class CHTScriptApiService {
     return this.initialized;
   }
 
-  private init() {
+  private async init() {
     this.watchChanges();
     this.userCtx = this.sessionService.userCtx();
-    return this.getSettings();
+    await Promise.all([ this.getSettings(), this.loadScripts() ]);
+  }
+
+  private async loadScripts() {
+    const request = this.http.get<String[]>('/extension-libs', { responseType: 'json' });
+    const extensionLibs = await lastValueFrom(request);
```

Comment:
Should we wrap this in a try-catch?  (Looks like we are doing that in the other places we call `http.get`....)

Reply:
Done!